src/main/kotlin/pt/up/fe/ni/website/backend/controller/AccountController.kt

```diff
@@ -29,22 +29,27 @@ class AccountController(private val service: AccountService) {
     @GetMapping("/{id}")
     fun getAccountById(@PathVariable id: Long) = service.getAccountById(id)
 
-    @PostMapping("/changePassword/{id}")
-    fun changePassword(@PathVariable id: Long, @RequestBody dto: ChangePasswordDto): Map<String, String> {
-        service.changePassword(id, dto)
-        return emptyMap()
+    @PostMapping("/new", consumes = ["multipart/form-data"])
+    fun createAccount(
+        @RequestPart account: CreateAccountDto,
```

Comment:
Just like with the accounts, the way to send files (in this case, an image) is to send 2 parts in the request: `{account: {...}, image: [file]`. `@RequestPart` reads the Dto from a property named `account` instead of reading the whole body